scripts/inactive_study.py

```diff
@@ -64,14 +67,23 @@
                        "for an OpenMC model divided into a number of layers")
 ap = ArgumentParser(description=program_description)
 
-ap.add_argument('-i', dest='script_name', type=str,
-                help='Name of the OpenMC python script to run (without .py extension)')
 ap.add_argument('-n-threads', dest='n_threads', type=int,
                 default=multiprocessing.cpu_count(), help='Number of threads to run Cardinal with')
-ap.add_argument('-input', dest='input_file', type=str,
+ap.add_argument('-i', dest='script_name', type=str, required=True,
+                help='Name of the OpenMC python script to run (without .py extension)')
+ap.add_argument('-input', dest='input_file', type=str, required=True,
                 help='Name of the Cardinal input file to run')
-
+ap.add_argument('--method', dest = 'method', choices =['all','half','window'], required=True,
+                help = "The type of way to detect steady state. Options are all, half, or window")
```

Comment:
Can you add some extra information on what the 3 options do?

Reply:
Do you want this extra info in the `help` doc string or in the comments at the top of the file?

Reply:
in the help doc string

Reply:
Let me know if the most recent push addressed your concerns.